packages/calypso-products/package.json

```diff
@@ -36,6 +36,7 @@
 	"homepage": "https://github.com/Automattic/wp-calypso/tree/HEAD/packages/calypso-products#readme",
 	"dependencies": {
 		"@automattic/calypso-config": "workspace:^",
+		"@automattic/onboarding": "workspace:^",
```

Comment:
@escapemanuele this might be causing the unit test failures. let's see if removing this dependency fixes the tests. we might have to go the other way with this (what we [discussed earlier](https://github.com/Automattic/wp-calypso/pull/75174#discussion_r1174437990) about creating `getNewsletterTagLine`, etc. VS having the smarter functions that do the conditioning inside `calypso-products`).